TCAT/Controllers/RouteDetail+ContentViewController.swift

```diff
@@ -452,7 +438,7 @@ class RouteDetailContentViewController: UIViewController {
     }
 
     /** Draw all waypoints initially for all paths in [Path] or [[CLLocationCoordinate2D]], plus fill bounds */
-    private func drawMapRoute() {
+    func drawMapRoute() {
```

Comment:
why did you is this function no longer private? is drawmaproute being used anywhere else?

Reply:
Good catch! I had been using it in a different file but found a better way. It's back now.